src/main/java/web/Calculator.java

```diff
@@ -0,0 +1,102 @@
+package web;
+
+import java.util.*;
+import java.util.regex.Pattern;
+
+class Calculator{
+    private static int calculate (int num1, int num2, char op) {
+        int number1 = num2;
+        int number2 = num1;
+        switch (op) {
+            case '*':
+                return number1 * number2;
+            case '/':
+                return number1 / number2;
+            case '+':
+                return number1 + number2;
+            case '-':
+                return number1 - number2;
+            default:
+                return 0;
+        }
+    }
+
+    private Map<String, String> variables;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4846841)